apps/docs/src/app/platform/component-docs/platform-icon-tab-bar/examples/platform-icon-tab-bar-configurable-paddings-example/platform-icon-tab-bar-configurable-paddings-example.component.ts

```diff
@@ -1,5 +1,5 @@
 import { Component } from '@angular/core';
-import { cloneDeep } from '@fundamental-ngx/core';
+import { cloneDeep } from 'lodash-es';
```

Comment:
please use deep import from lodash-es/cloneDeep

Reply:
I would also update blacklist of imports to restrict import from the root of lodash

Reply:
Tree-shaking works fine, when dependencies are imported from `lodash-es` directly. I've verified.

Reply:
> I would also update blacklist of imports to restrict import from the root of lodash
This is not easily doable at this point, I've created a separate ticket
https://github.com/SAP/fundamental-ngx/issues/7414